tests/graphql/errors/test-const-with-inline.graphql

```diff
@@ -3,8 +3,8 @@ schema @server @upstream(baseURL: "http://jsonplaceholder.typicode.com") {
   query: Query
 }
 
-type Query {
-  post: Post @http(path: "/posts/1") @inline(path: ["user", "name"])
+type Query @addField(name: "username", path: ["user", "name"]) {
```

Comment:
```suggestion
type Query @addField(name: "username", path: ["post","user", "name"]) {
```
Ideally this test should assert addfields and const in the path should give error

Reply:
I didn't notice this; I fixed it! nice catch!